src/arrayMethodEvery.js

```diff
@@ -5,7 +5,12 @@
  */
 function applyCustomEvery() {
   [].__proto__.every2 = function(callback) {
-    // write code here
+    for (let i = 0; i < this.length; i++) {
+      if (callback(this[i], i, this) === false) {
```

Comment:
такі перевірки ніколи не пишуть.
Використовуй логічний оператор `!` для перетворення в boolean
```suggestion
      if (!callback(this[i], i, this)) {
```